runtime/server_http_request.go

```diff
@@ -416,7 +433,7 @@ func (req *ServerHTTPRequest) ReadAll() ([]byte, bool) {
 		}
 		return nil, false
 	}
-
+	req.rawBody = rawBody
```

Comment:
it still needs to be idempotent.

`if req.rawBody != nil { ... }` 

We need a test that calling `ReadAll()` twice works.